File: src/main/java/forpleuvoir/hiirosakura/client/feature/task/TimeTaskData.java
package forpleuvoir.hiirosakura.client.feature.task;

/**
 * 定时任务 参数
 *
 * @author forpleuvoir
 * <p>#project_name hiirosakura
 * <p>#package forpleuvoir.hiirosakura.client.feature.task
 * <p>#class_name TimeTaskData
 * <p>#create_time 2021-07-23 16:25
 */
public record TimeTaskData(Integer startTime,Integer cycles,Integer cyclesTime,String name) {
}

File: src/main/kotlin/forpleuvoir/hiirosakura/client/feature/task/TimeTaskData.kt
package forpleuvoir.hiirosakura.client.feature.task

/**
 * 定时任务 参数
 *
 * @author forpleuvoir
 *
 * #project_name hiirosakura
 *
 * #package forpleuvoir.hiirosakura.client.feature.task
 *
 * #class_name TimeTaskData
 *
 * #create_time 2021-07-23 16:25
 */
class TimeTaskData(val name: String, val startTime: Int = 0, val cycles: Int = 1,val cyclesTime: Int = 0)